analyses/focal-cn-file-preparation/01-prepare-cn-file.R

```diff
@@ -53,17 +53,17 @@ if (!("AnnotationDbi" %in% installed.packages())) {
 #### Function ------------------------------------------------------------------
 
 process_annotate_overlaps <- function(cnv_df,
-                                      txdb_exons,
+                                      cds_object,
                                       filt_na_symbol = TRUE) {
   # This function takes a standardized data.frame that contains genomic range
-  # information and finds the overlaps with a TxDb object. It will discard
-  # Ensembl gene IDs with no gene symbol by default.
+  # information and finds the overlaps with a CDS GRanges object. It will
+  # discard Ensembl gene IDs with no gene symbol by default.
   #
   # Args:
   #   cnv_df: standardized data.frame that contains the segments used in the
   #           CNV caller
-  #   txdb_exons: exons to be merged with the CNV data.frame; output o
-  #               GenomicFeatures::exons
+  #   cds_object: coding sequences to be merged with the CNV data.frame; output
```

Comment:
What kind of object is this? May be better to be a bit more specific because I already forgot what kind of object it is. Is it a file path you supply or a data.frame or a GenomicRanges object?